src/coprocessor/metrics.rs

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use prometheus::*;
+use prometheus::{exponential_buckets, HistogramVec, IntCounterVec, IntGaugeVec};
```

Comment:
BTW I prefer to use * here. It is very unlikely to have collisions for these names.